DataFormats/L1Trigger/interface/BXVector.impl

```diff
@@ -1,18 +1,20 @@
 #include <vector>
 #include <iostream>
+#include <cassert>
 using namespace std;
```

Comment:
yes - indeed - thanks for noticing
@mulhearn

> On Apr 29, 2016, at 10:28 AM, Martin Grunewald <notifications@github.com> wrote:
> 
> In DataFormats/L1Trigger/interface/BXVector.impl:
> 
> > @@ -1,18 +1,20 @@
> >  #include <vector>
> >  #include <iostream>
> > +#include <cassert>
> >  using namespace std;
> 
> This line in a (file included in a) header file is problematic!
> @mulhearn @rekovic @davidlange6
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>